etc/configs/fake.json

```diff
@@ -12,6 +12,13 @@
                 "arm-model": "xArm6"
             }
         },
+        {
+            "type": "camera",
+            "model": "webcam",
+            "attributes": {},
+            "depends_on": [],
+            "name": "cam"
+        },
```

Comment:
📸🌟

Reply:
will this break for people who are testing on things without webcams? maybe we should make a separate fake config for easily testing camera

Reply:
Ah dang, yeah, I'm not sure what the behavior is if the device doesn't have a working webcam. Removing this for now and will ask around to see if this would break things for people.